docs/config.yml

```diff
@@ -75,8 +75,8 @@ menu:
   - identifier: "DDS"
     name: "Data Structures"
     weight: -300
-  - identifier: "deep"
```

Comment:
what's happening to the articles that remain under deep?  it doesn't look like there's a way to access all of them anymore with the new organization

Reply:
Correct, they're all hidden for now. Consensus seems to be that they're confusing.